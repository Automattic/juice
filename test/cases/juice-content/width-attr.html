<html>
<head>
<style>
    p, td.px, th.px, img.px {
        width: 200px;
    }
    td.percentage, th.percentage, img.percentage {
        width: 50%;
    }
    td.auto, th.auto, img.auto {
        width: auto;
    }
</style>
</head>
<body>
    <p>none</p>
    <table>
        <tr>
            <th class="px">
                header 1
            </th>
            <th class="percentage">
                header 2
            </th>
            <th class="auto">
                header 3
            </th>
        </tr>
        <tr>
            <td class="px">
                wide
            </td>
            <td class="percentage">
                wide
            </td>
            <td class="auto">
                wide
            </td>
        </tr>
    </table>
    <img src="#" alt="wide" class="px">
    <img src="#" alt="wide" class="percentage">
    <img src="#" alt="wide" class="auto">
</body>
</html>

