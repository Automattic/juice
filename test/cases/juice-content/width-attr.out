<html>
<head>

</head>
<body>
    <p style="width: 200px;">none</p>
    <table>
        <tr>
            <th class="px" style="width: 200px;" width="200">
                header 1
            </th>
            <th class="percentage" style="width: 50%;" width="50%">
                header 2
            </th>
            <th class="auto" style="width: auto;" width="auto">
                header 3
            </th>
        </tr>
        <tr>
            <td class="px" style="width: 200px;" width="200">
                wide
            </td>
            <td class="percentage" style="width: 50%;" width="50%">
                wide
            </td>
            <td class="auto" style="width: auto;" width="auto">
                wide
            </td>
        </tr>
    </table>
    <img src="#" alt="wide" class="px" style="width: 200px;" width="200">
    <img src="#" alt="wide" class="percentage" style="width: 50%;">
    <img src="#" alt="wide" class="auto" style="width: auto;" width="auto">
</body>
</html>
